l1-contracts/test/portals/UniswapPortal.t.sol

```diff
@@ -116,9 +134,78 @@ contract UniswapPortalTest is Test {
     outbox.sendL1Messages(entryKeys);
   }
 
-  function testSwap() public {
-    bytes32 daiWithdrawMsgKey = _createDaiWithdrawMessage(address(uniswapPortal));
-    bytes32 swapMsgKey = _createUniswapSwapMessage(aztecRecipient);
+  function testRevertIfWithdrawMessageHasNoDesignatedCaller() public {
```

Comment:
If you could make a small comment clarifying more, e.g., 
```solidity
Creates a withdraw transaction without a designated caller. Should fail when uniswap portal tries to consume as it since it tries using a designated caller.
```